src/test/java/redis/clients/jedis/tests/commands/PublishSubscribeCommandsTest.java

```diff
@@ -65,6 +65,27 @@ public void onUnsubscribe(String channel, int subscribedChannels) {
 
   @Test
   public void pubSubChannels() {
+    final List<String> expectedActiveChannels = Arrays.asList("testchan1", "testchan2",
+      "testchan3", "__sentinel__:hello");
```

Comment:
Since we are utilizing sentinel for the tests, there is an additional `__sentinel__:hello` channel that has subscribers:
```
Every Sentinel is subscribed to the Pub/Sub channel __sentinel__:hello of every master and replica, looking for unknown sentinels.
```
https://redis.io/topics/sentinel

Reply:
@sabbey37 Nice observation. Please add this in code comment as well.